src/test/java/api/loans/RenewByBarcodeTests.java

```diff
@@ -4,11 +4,25 @@
 import org.folio.circulation.support.http.client.Response;
 import org.folio.circulation.support.http.server.ValidationError;
 import org.hamcrest.Matcher;
+import org.junit.After;
+import org.junit.Before;
 
 import static api.support.matchers.ValidationErrorMatchers.hasMessage;
 import static api.support.matchers.ValidationErrorMatchers.hasParameter;
 
 public class RenewByBarcodeTests extends RenewalAPITests {
+
+  @Before
+  public void setupLoanType() {
+    noteTypeFixture.generalNoteType();
+  }
```

Comment:
The name refers to a loan type yet the implementation refers to a note type, could this be renamed to remove this discrepancy?